### PR TITLE
Various janitoring

### PR DESCRIPTION
**Pull request by akva2**
Mostly to satisfy static code analyzers.

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

File: test_util/EclRegressionTest.hpp [hunk 1]
```diff
@@ -79,7 +79,7 @@ class ECLRegressionTest: public ECLFilesComparator {
         this->printKeywordOnly = printArg;
     }
 
-    void compareSpesificKeyword(std::string keyword) {
+    void compareSpesificKeyword(const std::string& keyword) {
         this->spesificKeyword = keyword;
```
Comment (bska) [inline test_util/EclRegressionTest.hpp]: Does the static analyser accept the other way to write this, namely using `std::move`?
```C++
void compareSpesificKeyword(std::string keyword)
{
    this->spesificKeyword = std::move(kw);
}
```

Comment (akva2) [reply test_util/EclRegressionTest.hpp]: yes, rewritten to use move.

**Comment by akva2** [comment]
jenkins build this please

**Pull request merged**